addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua

```diff
@@ -283,37 +283,44 @@ function TOOL:Think()
   self:UpdateGhostWirejoystick( self.GhostEntity, self:GetOwner() )
 end
 
+local function trimTextEntry(pnText, sConvar, nLen)
+  local sTxt = pnSelf:GetText():Trim()
+        sTxt = sTxt:gsub("%s+", "0"):gsub("%W", "0")
+  if(sTxt:len() > nLen) then
+    sTxt = sTxt:sub(1, nLen)
+    ChangeTooltip(pnSelf)
```

Comment:
Where is this function defined?

Reply:
Here: https://wiki.facepunch.com/gmod/Global.ChangeTooltip

I am testing and going to update one more time soon

Reply:
The idea is to use the tooltip as a user hunt